### PR TITLE
Gemini prometheus metricsv2

### PR DESCRIPTION
**Pull request by aleksbykov**
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [ ] ~~All new and existing unit tests passed (`hydra unit-tests`)~~
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

File: sdcm/cluster.py [hunk 1]
```diff
@@ -3330,6 +3330,7 @@ def configure_scylla_monitoring(self, node, sct_metrics=True, alert_manager=True
                 templ_yaml = yaml.load(f, Loader=yaml.SafeLoader)  # to override avocado
                 self.log.debug("Configs %s" % templ_yaml)
             loader_targets_list = ["%s:9103" % n.ip_address for n in self.targets["loaders"].nodes]
+            gemini_loader_targets_list = ["%s:2112" % n.ip_address for n in self.targets["loaders"].nodes]
```
Comment (bentsi) [inline sdcm/cluster.py]: run this only if Gemini is intended to run

File: sdcm/cluster.py [hunk 2]
```diff
@@ -3339,6 +3340,8 @@ def remove_sct_metrics(x):
             scrape_configs = templ_yaml["scrape_configs"]
             scrape_configs.append(dict(job_name="stress_metrics", honor_labels=True,
                                        static_configs=[dict(targets=loader_targets_list)]))
+            scrape_configs.append(dict(job_name="gemini_metrics", honor_labels=True,
```
Comment (bentsi) [inline sdcm/cluster.py]: run this only if Gemini is intended to run

**Comment by bentsi** [comment]
please add a screenshot of the dashboard

File: sdcm/cluster.py (hunk 1 above)
Comment (aleksbykov) [reply sdcm/cluster.py]: fixed

File: sdcm/cluster.py (hunk 2 above)
Comment (aleksbykov) [reply sdcm/cluster.py]: fixed

**Comment by bentsi** [comment]
- why do we need to backport to 3.1? Does this PR fixes smth.?
- can you put an example dashboard screenshot?

**Comment by aleksbykov** [comment]
Screenshot from 2019-07-08 11-59-50

**Comment by aleksbykov** [comment]
1.1 Backport required, because we run the gemini and for 3.1 version. So i think this should be added also to branch-3.1.
1.2 PR fixed only the master dashboard, because it include the id of dashboard. what prevent from loading it to grafana.
screenshot added

**Pull request merged**